geometry_msgs/PoseStamped pose_right
---
bool succeeded
